src/vanilla/Templates/MethodGroupTemplate.cshtml

```diff
@@ -11,14 +11,18 @@ import * as msRest from "ms-rest-js";
 {
 @:import * as Models from "../models";
 }
-import * as Mappers from "../models/mappers";
+import * as Mappers from "../models/@(Model.TypeName.ToCamelCase())Mappers";
 import { @(Model.CodeModelTS.ContextName) } from "../@(Model.CodeModelTS.ContextName.ToCamelCase())";
 @EmptyLine
 const WebResource = msRest.WebResource;
 @EmptyLine
 /** Class representing a @(Model.TypeName). */
 export class @(Model.TypeName) {
   private readonly client: @(Model.CodeModelTS.ContextName);
+@{
+  var serializerParams = Model.CodeModel.ShouldGenerateXmlSerialization ? "Mappers, true" : "Mappers";
```

Comment:
I've seen this a few times now, too. Can you encapsulate it as a property/method on MethodGroupTS too?